src/core/var/rsingleTag.js

```diff
@@ -1,5 +1,5 @@
 define( function() {
 
 	// Match a standalone tag
-	return ( /^<([\w-]+)\s*\/?>(?:<\/\1>|)$/ );
+	return ( /^<([a-zA-Z][^\s\/\0>]*)\s*\/?>(?:<\/\1>|)$/ );
```

Comment:
Sure. I'll apply the aforementioned changes.

Regarding making it case insensitive, I considered that and did a quick research about it, but all the pieces of information I found said that the performance would be considerably lower than adding both [a-zA-Z]. Just let me know if you'd rather taking the chance and making it case insensitive.